doc/faq.rst

```diff
@@ -26,6 +26,21 @@ See :ref:`contributing`. Before wanting to add a new algorithm, which is
 usually a major and lengthy undertaking, it is recommended to start with :ref:`known
 issues <easy_issues>`.
 
+What's the best way to ask questions about scikit-learn?
+--------------------------------------------------------------
+The quickest way to get replies to your questions is posting on 
+`Stack Overflow <http://stackoverflow.com>`_ or 
+`Cross Validated <http://stats.stackexchange.com>`_ 
+ with the "scikit-learn" or "machine learning" tags. These sites may already have an answer
```

Comment:
minor nitpick, but stackexchange doesn't allow spaces in tag names. Thus the common tag for ML is "machine-learning"